### PR TITLE
Add ability to disable alpha channel. 

### PR DESCRIPTION
**Pull request by przemyslaw-szurmak**
Useful when doing upload to AppStore Connect

### CODE REVIEW

**Comment by NSAntoine** [comment]
Looks good to me!

**Comment by NSAntoine** [comment]
could you make this an optional argument though? And set the default arg to nil

File: Sources/AssetCatalogWrapper/AssetCatalogWrapper.swift [hunk 1]
```diff
@@ -500,7 +500,11 @@ public extension CUICatalog {
             else {
                 throw _Errors.failedToEditItem()
             }
-            
+
+            if isAlphaAllowed == false {
```
Comment (NSAntoine) [inline Sources/AssetCatalogWrapper/AssetCatalogWrapper.swift]: if !isAlphaAllowed instead

Comment (przemyslaw-szurmak) [reply Sources/AssetCatalogWrapper/AssetCatalogWrapper.swift]: fixed

**Comment by przemyslaw-szurmak** [comment]
> could you make this an optional argument though? And set the default arg to nil

I guess you meant to make default = `true` which is what I just did

**Comment by NSAntoine** [comment]
Oh I see, sorry I read this first on mobile, alright merging now, thank you!

**Comment by przemyslaw-szurmak** [comment]
> Oh I see, sorry I read this first on mobile, alright merging now, thank you!

Great, thanks! 

btw. thanks for all the effort to build this lib, very helpful stuff :)

**Pull request merged**